src/test/resources/jenkins/plugins/git/tool-casc.yaml

```diff
@@ -0,0 +1,20 @@
+tool:
+  git:
+    installations:
+      - home: "git"
```

Comment:
**Summary**: Ignore this suggestion for now, because it should not work until a new beta git client plugin has been released from the master branch 

I would prefer that the configuration as code test example include JGit and JGit Apache since they are now working and reliable.  Likewise I'd prefer that one or more of the examples refer to JGit or JGit Apache.

Though, now that I think about it, that should probably be added in a later change, outside the context of this pull request.  The JGit and JGit Apache change requires a new beta version of git client plugin released on the master branch and that has not happened yet.
```suggestion
      - name: "jgit"
      - name: "jgitapache"
      - home: "git"
```

Reply:
Ignoring for now then. It was intended not to tested them because they are part of the git-client and I thought they were already tested in that plugin.